demos/prefsFramework/index.html

```diff
@@ -418,12 +418,14 @@ <h1 class="ocean">Climate Change Impacts</h1>
 
                         <img class="icon" style="padding-right: 5px; float: right;" src="images/theme_climate_90.gif" alt="Oceans and Coasts" height="90" width="90">
                         <aside id="left_box">
-
-                            <div class="left_tab"></div>
-                            <div class="middle_tab">
-                                <h2 class="h1_middle_text fl-inverted-color">Resources</h2>
+                            <div class="fl-inverted-color">
+                                <div class="left_tab"></div>
+                                <div class="middle_tab">
+                                    <h2 class="h1_middle_text fl-inverted-color">Resources</h2>
```

Comment:
With the new `fl-inverted-color` container `div`, did we also want to keep the `fl-inverted-color` class here on the `h2`? We now have that class in 2 places: on a container and on the element.